specs/docker/list_docker_images_from_local.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: List docker images from local system
+command: docker image ls 
+tags:
+  - docker
+description: Lists all docker images currently stored on your system
+arguments: []
```

Comment:
@guel-codes i think this is failing because of "arguments: []"
Can you remove this line entirely, and I'll try re-running the GitHub action

Reply:
@elviskahoro should be good now, sorry about that

Reply:
Gonna rerun the workflow and see if it works!